refresh/fromswagger/swaggerize.js

```diff
@@ -70,12 +69,10 @@ function loadAsync (memfs, path) {
 
 function ensureValid (api, apiPath) {
```

Comment:
Since this function now returns a promise, it might be nice to rename it to `ensureValidAsync`.